src/zugferd10/Builder.php

```diff
@@ -25,7 +24,8 @@ public static function create(): Builder
     {
         $serializer = SerializerBuilder::create()
             ->setDebug(true)
-            ->build();
+            ->build()
```

Comment:
I've not used php-cs-fixer before, I'm not satisfied with this result…
`./vendor/bin/php-cs-fixer fix ./src --config .php_cs.dist`